prepare/acroname.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import optparse
```

Comment:
what about adding a shebang? We should be able to run the program w/o python prefix later on